### DISCUSSION TITLE
Bug: IntelliJ plugin update always fails

### DISCUSSION THREAD

**Discussion by Captain-Chaos** [community/community#17259] [Copilot Conversations]
The last few weeks, whenever Copilot says there is an update for my IntelliJ plugin and I click the link to install it, it works for a while and then consistently fails with:

> The update of GitHub Copilot failed. Please try again later. Error message: Plugin 'GitHub Copilot' was not successfully downloaded

[idea.log](https://github.com/github/feedback/files/8769867/idea.log)

**Comment by jansorg** [comment]
Thanks for reporting! Does the update by the IDE itself (e.g. at `Preferences > Plugins > Installed`) work for you?

**Reply by Captain-Chaos** [reply]
Yes, updating with IntelliJ itself consistently does work.

**Reply by flyingace** [reply]
Worked with WebStorm as well.

**Comment by feoc10** [comment]
This issue has happened with me as well. 
With last two versions (1.1.23.1570 and 1.1.22.1530), the Copilot can't give any suggestion because every time I change my code, it indicates that is having some temporary issues.

**Reply by jansorg** [reply]
@feoc10 That's likely a different problem. Could you post a screenshot, please?

**Reply by feoc10** [reply]
![image](https://user-images.githubusercontent.com/29875672/170266513-5928891f-e505-4a77-90dd-3ba7a73d26b2.png)

When a change the code, the icon turns orange and don't give me any tip

**Reply by feoc10** [reply]
If I use older versions (as 1.1.20.1417), it works perfect.

**Reply by Captain-Chaos** [reply]
My feedback is specifically about the plugin update. This is a different problem; could you file a separate issue for this please?

**Reply by michaelhachen** [reply]
Same here... I'm unable to update the plugin.
